mne/surface.py

```diff
@@ -1667,19 +1667,27 @@ def marching_cubes(image, level, smooth=0):
     ----------
     image : ndarray
         The image to compute marching cubes with.
-    level : float
+    level : float | int | list
         The contour value to search for isosurfaces in ``image``.
+        Can also be a list of integers to use a discrete marching cubes
+        algorithm suitable for labeled volumes (e.g., ``'aseg.mgz'``).
     %(smooth)s
 
     Returns
     -------
-    verts : ndarray
-        The spatial coordinates for unique mesh vertices.
-    triangles : ndarray
-        The locations of connections between ``verts`` to form faces.
+    out : list | tuple
```

Comment:
This could be called polydata to be more informative

Reply:
It's not polydata, that's a VTK concept. The output is really either a 2-tuple of ndarray or a list of 2-tuples of ndarray